Create a Main Menu Scene for the Game

cailafinn opened the pull request:
**Description of work.**
Created a main menu, contains a how to play section and level select. 
Currently only level one is available and it simply loads the "Hall" scene.
Also imports TextMeshPro

**To test:**
1. Open the MainMenu scene and play it
2. Check all the buttons work properly
3. Check the scaling is correct 
4. Check the scene loads properly (note: you may need to add both the menu and the hall scenes in `File -> Build Settings -> Scenes in Build`

Code review required on `MainMenuController.cs`

Fixes #45
<!-- alternative
*There is no associated issue.*
-->

Commit:
Add basic main menu RE #45

Commit:
Create basic main menu RE #45

Commit:
Add How to Play sub-menu RE #45

Commit:
Switch to a more interesting font RE #45

Commit:
Made backgroud black RE #45

Commit:
Create level select menu RE #45

Alice-Jaffray reviewed (changes requested):
The code looks good but the main menu itself is looking jumbled on my screen (I checked the scale etc) 
![Screenshot 2021-01-07 at 15 46 15](https://user-images.githubusercontent.com/47036197/103912910-ccee8b80-50ff-11eb-90dd-a8b7de36e7e9.png)

Commit:
Assign level to main menu script variable RE #45

Variable was left unassigned, meaning it was not hidden when the game is
started. This has now been amended.

Alice-Jaffray reviewed (commented):
Ok, this might be a personal opinion but I don't think the titles should overlap. Also, I think it makes more sense in terms of HCI to have the grey background for the available buttons and black background for those that aren't. What do you think?

Commit:
Adjust colour of buttons for clarity RE #45

Alice-Jaffray reviewed (approved):
this is fab-u-lous darling
